cfe_internal/enterprise/CFE_knowledge.cf

```diff
@@ -235,11 +235,11 @@ bundle agent cfe_internal_setup_knowledge
         file_select => cfe_internal_docroot_api_static_async_query_status_status_perms,
         perms => mog("0660", "root", $(def.cf_apache_group) );
 
-      "$(sys.workdir)/httpd/logs/application/." -> { "ENT-7731", "ENT-2758" }
-      comment => "Ensure permissions for $(sys.workdir)/httpd/logs/application/.",
-      handle => "cfe_internal_setup_knowledge_files_httpd_application_log_dir",
-      create => "true",
-      perms => mog("0640", $(def.cf_apache_user), $(def.cf_apache_group));
+      "$(sys.workdir)/httpd/logs/application/." -> { "ENT-7731", "ENT-2758", "ENT-8908", "CFE-951" }
```

Comment:
Wow, that's a lot of tickets for one directory :)

Reply:
it's been a directory of much discussion :p